soccer/planning/MultiRobotPathPlanner.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <planning/MotionCommand.hpp>
+#include <planning/MotionConstraints.hpp>
+#include <planning/MotionInstant.hpp>
+#include <planning/Path.hpp>
+
+#include <map>
+#include <memory>
+
+namespace Planning {
+
+/// The PlanRequest encapsulates all information that the planner needs to know
+/// about an individual robot in order to generate a path for it.
+struct PlanRequest {
```

Comment:
If `MultiRobotPathPlanner` is the only place it's used, I think it's probably fine to keep it here.  If we want to make `SingleRobotPathPlanner` take this in rather than having separate arguments, then it would probably be best in its own file.